### ISSUE TITLE
Create integration test for CLI usage

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/hoodie#530]
This is a note for later (once #529 is merged). I want to make sure we have test covering the CLI usage, so we catch issues like these before we release in future: https://github.com/hoodiehq/hoodie/pull/529#issuecomment-233223493

**Comment by gr2m** [comment]
we don’t have a CLI any more so this is no longer relevant right now
